src/EditorFeatures/Test/BraceHighlighting/AbstractBraceHighlightingTests.cs

```diff
@@ -16,9 +17,9 @@ namespace Microsoft.CodeAnalysis.Editor.UnitTests.BraceHighlighting
 {
     public abstract class AbstractBraceHighlightingTests
     {
-        protected async Task TestBraceHighlightingAsync(string markup)
+        protected async Task TestBraceHighlightingAsync(string markup, CSharpParseOptions options = null)
```

Comment:
`AbstractBraceHighlightingTests` is only referenced in C# projects as it is. But I see your point, so I'll change to `ParseOptions` here.